### PR TITLE
add more information on code styles to PHP guide

### PR DESCRIPTION
**Pull request by bshaffer**

### CODE REVIEW

File: content/en/_index.md [hunk 1]
```diff
@@ -877,8 +877,14 @@ try {
 ### Linting {#linting}
 
 Our code snippets should adhere to a style used by the language communities.
-We prefer to enforce style using standard linters that are popular in
-the community.
+We follow the [PSR-1][psr-1] and [PSR-2][psr-2] community coding standards, with
```
Comment (grayside) [inline content/en/_index.md:880]: This section is cross-language. Could you add a  `content-tabpane` here similar to what we're using to wrap the text under **Language-specific practices** below?

**Review by grayside — CHANGES_REQUESTED** [review]
Thanks for adding this detail Brent!

File: content/en/_index.md (hunk 1 above)
Comment (bshaffer) [reply content/en/_index.md]: OK, I think I did it right

File: content/en/_index.md [hunk 2]
```diff
@@ -967,6 +967,17 @@ public static void exampleSnippet(String projectId, String filePath) {
...
+
+We follow the [PSR-1][psr-1] and [PSR-2][psr-2] community coding standards, with
+one notable exception:
+   
+1. The function names for sample are in `snake_case` rather than `camelCase`,
+   in order to differentiate them from class methods.
+   
+[psr-1]: https://www.php-fig.org/psr/psr-1/
+[psr-2]: https://www.php-fig.org/psr/psr-2/#1-overview
+
```
Comment (grayside) [inline content/en/_index.md:980]: I think I was unclear. Please move this to line 882 with a new content-tabpane under the exising linting guideline.

**Review by grayside — CHANGES_REQUESTED** [review]

File: content/en/_index.md (hunk 2 above)
Comment (bshaffer) [reply content/en/_index.md:980]: I'm not familiar with the syntax being used here, or how to preview the changes. I am also sure what you're asking. This is a very minor and low priority change and we have it documented internally so I am closing this instead.

**Pull request closed without merging**